app/graphql/types/fees/object.rb

```diff
@@ -38,6 +41,16 @@ def item_type
       def applied_taxes
         object.applied_taxes.order(tax_rate: :desc)
       end
+
+      def adjusted_fee
+        object.adjusted_fee.present?
+      end
+
+      def adjusted_fee_type
+        return nil unless object.adjusted_fee.present?
```

Comment:
Rails/Blank: Use `if object.adjusted_fee.blank?` instead of `unless object.adjusted_fee.present?`.